src/app/components/ATIAnalytics/params/genericPage/buildParams.ts

```diff
@@ -11,27 +11,35 @@ export const buildPageATIParams = ({
   const { atiAnalyticsAppName, atiAnalyticsProducerId, service, lang } =
     serviceContext;
   const {
-    analytics: {
-      contentId,
-      contentType,
-      pageIdentifier,
-      timePublished,
-      timeUpdated,
-    } = {},
-    title: pageTitle,
+    contentId,
+    contentType,
+    pageIdentifier,
+    timePublished,
+    timeUpdated,
+    pageTitle,
   } = atiData;
+  // const {
+  //   atiAnalytics: {
+  //     contentId,
+  //     contentType,
+  //     pageIdentifier,
+  //     timePublished,
+  //     timeUpdated,
+  //     pageTitle,
+  //   } = {},
+  // } = atiData;
   return {
     appName: atiAnalyticsAppName,
     contentId,
     producerId: atiAnalyticsProducerId,
     contentType,
     pageIdentifier,
-    pageTitle,
     platform,
     statsDestination,
     service,
     timePublished,
     timeUpdated,
+    pageTitle,
```

Comment:
Why this now not alphabetical?

Reply:
An oversight in the renaming of the `title` alias.
This has been changed.